### ISSUE TITLE
Add notebook on OpenQasm and conversions

### ISSUE THREAD

**Issue by nathanshammah** [qutip/qutip-notebooks#110]
It would be nice to have a simple and short notebook to show how to export and import between OpenQasm and `qutip.qip.circuit.QubitCircuit`, once [#1291](https://github.com/qutip/qutip/pull/1291) is merged. The same could apply to Cirq and qiksit (if applicable) in similar notebooks or the same.
